crates/sui-faucet/src/server.rs

```diff
@@ -285,3 +553,229 @@ async fn handle_error(error: BoxError) -> impl IntoResponse {
         Cow::from(format!("Unhandled internal error: {}", error)),
     )
 }
+
+/// Format seconds to human readable format.
+fn secs_to_human_readable(total_seconds: u64) -> String {
```

Comment:
This function would need to accept a `Duration` for my earlier code suggestion to work.